src/main/resources/assets/tfc/patchouli_books/book/en_us/entries/advanced_metalworking/iron.json

```
@@ -46,8 +46,9 @@
 			"text": "The block immediately above that should also be empty and surrounded on all four sides by stone. Basically it's a two-block high stone chimney connected to the back of the Bloomery Block. To let the bloomery store more ore, add blocks to grow the chimney higher.$(br2)$(thing)Flip page to see diagram$()"
 		},
 		{
-			"type": "text",
-			"text": "insert bloomery multiblock here"
+		  "type": "multiblock",
+		  "multiblock_id": "tfc:bloomery",
+		  "text": "Stuff n things"
```

```suggestion
		  "text": "The smallest possible bloomery multiblock."
```
May not fit, need to check in game.